soroban-sdk/src/accounts.rs

```diff
@@ -265,6 +273,18 @@ impl AccountId {
     }
 }
 
+#[cfg(feature = "testutils")]
+#[cfg_attr(feature = "docs", doc(cfg(feature = "testutils")))]
+impl crate::testutils::AccountId for AccountId {
+    fn random(env: &Env) -> AccountId {
```

Comment:
Turns out we already had afunction for this, `Accounts::generate`, so I moved that logic into a random functions on `AccountId`, and that replaces the `crate::testutils::random_account_id`.